src/agents/interactive_parameters.jl

```diff
@@ -42,64 +42,52 @@ data plots when resetting, for visual guidance.
 * `when = true`: When to perform data collection, as in `Agents.run!`.
 * `spu = 1:100`: Values that the "spu" slider will obtain.
 """
-function abm_data_exploration(
-        model, agent_step!, model_step!, params = Dict();
-        mdata = nothing,
-        adata = nothing,
-        alabels = nothing,
-        mlabels = nothing,
-        when = true,
-        spu = 1:100,
-        colorscheme = JULIADYNAMICS_COLORS,
-        kwargs...
-    )
+function abm_data_exploration(model, agent_step!, model_step!, params = Dict();
+    adata = nothing, mdata = nothing, alabels = nothing, mlabels = nothing, when = true,
+    spu = 1:100, colorscheme = JULIADYNAMICS_COLORS, kwargs...)
 
     # Initialize main layout
-    fig, abmstepper = abm_plot(model;
-        resolution=(1600,800), colorscheme, kwargs...
-    )
+    fig, abmstepper = abm_plot(model; resolution = (1600, 800), colorscheme, kwargs...)
 
     # Initialize agent and model dataframes
     adf = Agents.init_agent_dataframe(model, adata)
     mdf = Agents.init_model_dataframe(model, mdata)
 
     # Initialize data plots and define button behavior
-    abm_data_exploration!(fig, abmstepper, model, agent_step!, model_step!, params; 
-        spu, when, mdata, adata, alabels, mlabels, adf, mdf, colorscheme
-    )
+    abm_data_exploration!(fig, abmstepper, model, agent_step!, model_step!, params;
+        spu, when, adata, mdata, alabels, mlabels, adf, mdf, colorscheme)
```

Comment:
I don't understand why such changes are being made. It makes reviewing the PR hard without reason.

Reply:
Yes, sorry. This is mostly due to the automatic formatting tools in VS Code always messing with things. I think I need to turn it off because it is honestly just annoying to have it even intervene in things related to coding style (e.g. exchanging `in` with `=`). I will clean it up again.

Reply:
In addition the styling that is currently done was also done with a linter from @Libbum which i prefer more than yours. No offense I hope :D

Reply:
Which one does @Libbum use? I have no strong opinion on this topic, so no offense taken at all.

Reply:
Julia's VS Code extension uses https://github.com/julia-vscode/DocumentFormat.jl by default.
There's also an extension for https://github.com/domluna/JuliaFormatter.jl as it seems.
Don't know if there are any other relevant formatters.

Reply:
Sorry again about the formatting issues @Datseris. If you hide whitespace changes on github's diff view, it's a bit more bearable to look at. Anyways, I've highlighted the relevant code regarding the new "clear all" button below. This is the core of this PR.